pkg/engine/lifecycletest/pulumi_test.go

```diff
@@ -5498,79 +5498,6 @@ func TestConstructCallDependencyDedeuplication(t *testing.T) {
 	})
 }
 
-func TestContinueOnError(t *testing.T) {
```

Comment:
This and the other continue on error tests now live in `continue_on_error_test.go`, as `pulumi_test.go` feels a little bit too much like kitchen sink of tests.